sdk/storage/storage-blob/src/RetryPolicyFactory.ts

```diff
@@ -71,31 +74,39 @@ export interface RetryOptions {
    * {@link https://docs.microsoft.com/en-us/azure/storage/common/storage-designing-ha-apps-with-ragrs}
    *
    * @type {string}
-   * @memberof RetryOptions
+   * @memberof StorageRetryOptions
    */
   readonly secondaryHost?: string;
 }
 
 /**
- * RetryPolicyFactory is a factory class helping generating RetryPolicy objects.
+ * StorageRetryPolicyFactory is a factory class helping generating StorageRetryPolicy objects.
  *
  * @export
- * @class RetryPolicyFactory
+ * @class StorageRetryPolicyFactory
  * @implements {RequestPolicyFactory}
  */
-export class RetryPolicyFactory implements RequestPolicyFactory {
-  private retryOptions?: RetryOptions;
+export class StorageRetryPolicyFactory implements RequestPolicyFactory {
```

Comment:
>StorageRetryPolicyFactory [](start = 13, length = 25)

Rename the file name too. Like StorageRetryPolicyFactory.ts

We have practices to align the file name with the class name in it.

Reply:
@XiaoningLiu The policy files have been renamed too. Please take another look